src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -167,6 +183,51 @@ class AccessibleView extends Disposable {
 		this._currentProvider.next?.();
 	}
 
+	goToSymbol(): void {
+		this._instantiationService.createInstance(AccessibleViewSymbolQuickPick, this).show(this._currentProvider!);
+	}
+
+	getSymbols(): IAccessibleViewSymbol[] | undefined {
+		if (!this._currentProvider) {
+			return;
+		}
+		const tokens = this._currentProvider.options.language && this._currentProvider.options.language !== 'markdown' ? this._currentProvider.getSymbols?.() : marked.lexer(this._currentProvider.provideContent());
+		if (!tokens) {
+			return;
+		}
+		const symbols: IAccessibleViewSymbol[] = [];
+		for (const token of tokens) {
+			let label: string | undefined = undefined;
+			if ('type' in token) {
+				switch (token.type) {
+					case 'heading':
+					case 'paragraph':
+					case 'code':
+						label = token.text;
+						break;
+					case 'list':
+						label = token.items?.map(i => i.text).join(', ');
+						break;
+				}
+			} else {
+				label = token.label;
+			}
+			if (label) {
+				symbols.push({ info: label, label: localize('symbolLabel', "({0}) {1}", token.type, label), ariaLabel: localize('symbolLabelAria', "({0}) {1}", token.type, label) });
```

Comment:
the quick pick's aria label isn't read by the screen reader, so unfortunately, we have to directly provide symbol info here